Update view-and-configure-usage-analytics-reports.md (#2987)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointServer/administration/view-and-configure-usage-analytics-reports.md

The usage analytics reports show statistics of how visitors have interacted with the content on your website, for example which item is viewed the most. The reports are broken down per usage event. By default, this means that you'll be able to see:
  
- The number of *views* for an item 
    
- The number of *recommendations displayed* per item 
    
- The number of *recommendations clicked* per item 
    
SharePoint Server creates two usage analytics reports:
  
- *Most Popular Items* shows usage event statistics for all items in a library or list
    

> - *Popularity Trends* shows usage event statistics for an item, a site or a site collection

Reviewer: ```suggestion
- *Popularity Trends* shows usage event statistics for an item, a site, or a site collection
```

  
![Ret On Sale Query](../media/OTCSP_RetOnSaleQuer.png)
  
To see *Views* usage event statistics for items that are on sale, on the **Most Popular Items** page, select **Most Views**, and enter *RetOnSale=True*. 
  
![Ret On Sale Report](../media/OTCSP_RetOnSaleReport.png)
  
And just like that, only items that are on sale are shown.
  
You can also narrow usage event statistics to a specific category, for example all *Laptops*. In our Contoso scenario, the *Item Category* site column is used to specify the category for each item. 
  
![Item Category Site Column](../media/OTCSP_ItemCategorySiteColumn.png)
  
The managed property name of the *Item Category* column is *owstaxIdProductCatalogItemCategory*. The *owstaxIdProductCatalogItemCategory* managed property is *queryable*, which means that we can use it to narrow the Most Popular Items report. 
  
![owstax Id Product Catalog Item Category](../media/OTCSP_owstaxIdProductCatalogItemCategory.png)
  
To see which laptops are viewed most by visitors, on the **Most Popular Items** page, select **Most Views**, and enter *owstaxIdProductCatalogItemCategory:Laptops*. 
  
![Laptop Views](../media/OTCSP_LaptopViews.png)
  

> By narrowing the *Views* usage event statistics to a specific category, we can verify that the **Popular Items Web Part** is working correctly on our publishing site. For example, on the *Laptops* page, notice that the three items that are displayed in the **Popular Items Web Part** are the same that are shown to have the most *Views* in the Most Popular Items report that we just narrowed to *Laptops*. 

Reviewer: ```suggestion
By narrowing the *Views* usage event statistics to a specific category, we can verify that the **Popular Items Web Part** is working correctly on our publishing site. For example, on the *Laptops* page, notice that the three items that are displayed in the **Popular Items Web Part** are the same as those shown to have the most *Views* in the Most Popular Items report that we just narrowed to *Laptops*. 
```

## About the TailTrimming parameter
<a name="BKMK_AbouttheTailTrimmingParameter"> </a>

While we're on the subject of usage analytics reports, one usage event parameter that is worth mentioning is *TailTrimming*. This parameter specifies when the numbers that are shown in the usage analytics reports should be updated. 
  
> [!IMPORTANT]
> This parameter does not affect how recommendations are calculated. 
  
For example, for the Views usage event the *TailTrimming* parameter is by default set to 3. 
  
![Tail Trimming 3](../media/OTCSP_TailTrimming3.png)
  

> This means that the usage analytics reports will be updated for an item when the item was viewed a minimum three times within the past 24 hours. For example, if the item *Fabrikam Laptop16 M6000* was viewed two times within the past 24 hours, these two views won't be displayed in the usage analytics reports. If within the next 24 hours the *Fabrikam Laptop16 M6000* item is viewed 4 times, the usage analytics report will be updated with 4 views. 

Reviewer: ```suggestion
This means that the usage analytics reports will be updated for an item when the item was viewed a minimum of three times within the past 24 hours. For example, if the item *Fabrikam Laptop16 M6000* was viewed two times within the past 24 hours, these two views won't be displayed in the usage analytics reports. If within the next 24 hours the *Fabrikam Laptop16 M6000* item is viewed four times, the usage analytics report will be updated with four views. 
```
